README.md

```diff
@@ -1,3 +1,3 @@
 # react-page-click [![MOVED](https://img.shields.io/badge/status-MOVED-blue.svg?style=flat-square)](https://github.com/nkbt/react-works/tree/master/packages/page-click)
 
-All code moved to https://github.com/nkbt/react-works/tree/master/packages/page-click
+Dear contributors i would like to inform you that all the codes of this vary respository have been moved to https://github.com/nkbt/react-works/tree/master/packages/page-click
```

Comment:
Sorry, mate, I found at least 3 grammar mistakes in this one sentence, which is especially funny considering PR title.

To the main point - would really appreciate at least some elaboration on why this change was necessary.

Cheers 👍